samples/subsys/debug/gdbstub/pytest/test_gdbstub.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import subprocess
+from twister_harness import DeviceAdapter
+import sys
+import logging
+import shlex
+
+ZEPHYR_BASE = os.getenv("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts", "pylib", "twister"))
+from twisterlib.cmakecache import CMakeCache
+
+logger = logging.getLogger(__name__)
+
+
+def test_gdbstub(dut: DeviceAdapter):
+    """
+    Test description should be added here (will be displayed with pytest -v --co)
+    """
```

Comment:
It was just an example, description of test should be updated

Reply:
:)